nixos/modules/services/hardware/fw-fanctrl.nix

```
@@ -0,0 +1,129 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.hardware.fw-fanctrl;
+  defaultConfig = builtins.fromJSON (
+    builtins.readFile "${pkgs.fw-fanctrl}/share/fw-fanctrl/config.json"
+  );
```

```suggestion
  defaultConfig = lib.importJSON "${pkgs.fw-fanctrl}/share/fw-fanctrl/config.json"
```
that triggers IFD if I am not mistaken